image/image_linux.go

```diff
@@ -163,8 +163,8 @@ func Prepare(opts *Options) error {
 		if model.Classic() {
 			return fmt.Errorf("cannot preseed the image for a classic model")
 		}
-		if model.Base() != "core20" {
-			return fmt.Errorf("cannot preseed the image for a model other than core20")
+		if model.Base() == "core" || model.Base() == "core18" {
```

Comment:
We can have bases that are not coreXX (bare, or other non-canonical bases) for which we should return error, so it would be better to allow only for core20+.

Reply:
This should be fixed. I have forced-push because I changed everything.